rest/taskana-rest-spring/src/main/java/pro/taskana/rest/WorkbasketDefinitionController.java

```diff
@@ -56,17 +57,23 @@
 
     @GetMapping
     @Transactional(readOnly = true, rollbackFor = Exception.class)
-    public ResponseEntity<List<WorkbasketSummary>> exportWorkbaskets(@RequestParam(required = false) String domain) {
+    public ResponseEntity<List<WorkbasketDefinition>> exportWorkbaskets(@RequestParam(required = false) String domain)
+        throws NotAuthorizedException, WorkbasketNotFoundException {
         LOGGER.debug("Entry to exportWorkbaskets(domain= {})", domain);
         WorkbasketQuery workbasketQuery = workbasketService.createWorkbasketQuery();
         List<WorkbasketSummary> workbasketSummaryList = domain != null
             ? workbasketQuery.domainIn(domain).list()
             : workbasketQuery.list();
+        List<WorkbasketDefinition> basketExports = new ArrayList<>();
```

Comment:
This was removed in a merge, it was an error. I have redo this code and check if there were more conflicts wrongly resolved.